framework/src/modules/auth/module.ts

```diff
@@ -33,10 +43,144 @@ export class AuthModule extends BaseModule {
 	public configSchema = configSchema;
 	public commands = [new RegisterMultisignatureCommand(this.id)];
 
-	// eslint-disable-next-line @typescript-eslint/require-await
-	public async verifyTransaction(_context: TransactionVerifyContext): Promise<VerificationResult> {
+	public async verifyTransaction(context: TransactionVerifyContext): Promise<VerificationResult> {
+		const { transaction, networkIdentifier } = context;
+		const store = context.getStore(this.id, STORE_PREFIX_AUTH);
+		const senderAccount = await store.getWithSchema<AuthAccount>(
+			transaction.senderAddress,
+			authAccountSchema,
+		);
+
+		if (transaction.nonce < senderAccount.nonce) {
+			return {
+				status: VerifyStatus.FAIL,
+				error: new InvalidNonceError(
+					`Transaction with id:${transaction.id.toString('hex')} nonce is lower than account nonce`,
+					transaction.nonce,
+					senderAccount.nonce,
+				),
+			};
+		}
+		const transactionBytes = transaction.getSigningBytes();
+
+		if (
+			transaction.moduleID === this.id &&
+			this.commands.find(cmd => cmd.id === transaction.commandID)
```

Comment:
I think second condition will be invalid if we add more commands. Probably better to specifically check if it's 'registerMultisignatureGroup' command